mobile/lib/modules/search/providers/search_result_page.provider.dart

```diff
@@ -61,12 +62,11 @@ final searchResultGroupByDateTimeProvider = StateProvider((ref) {
   var assets = ref.watch(searchResultPageProvider).searchResult;
 
   assets.sortByCompare<DateTime>(
-    (e) => DateTime.parse(e.createdAt),
+    (e) => e.createdAt,
```

Comment:
I am wondering here what would happen if the parsing failed in the `Asset` model?

Reply:
The same as before my changes. A `FormatException` is thrown.